### PR TITLE
[16.0] [MIG] procurement_auto_create_group

### PR DESCRIPTION
**Pull request by lmignon**

### CODE REVIEW

**Comment by lmignon** [comment]
cc @rousseldenis @jbaudoux

File: procurement_auto_create_group/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+* Jordi Ballester <jordi.ballester@forgeflow.com>
+* Lois Rilo <lois.rilo@forgeflow.com>
+* Héctor Villarreal Ortega <hector.villarreal@forgeflow.com>
+* Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+* Bernat Puig <bernat.puig@forgeflow.com>
```
Comment (rousseldenis) [inline procurement_auto_create_group/readme/CONTRIBUTORS.rst:5]: Put yourself in contributors

File: procurement_auto_create_group/tests/test_auto_create.py [hunk 2]
```diff
@@ -0,0 +1,237 @@
+# Copyright 2017-2020 ForgeFlow, S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestProcurementAutoCreateGroup(TransactionCase):
+    def setUp(self):
```
Comment (rousseldenis) [inline procurement_auto_create_group/tests/test_auto_create.py]: Could you switch to classmethod ?

Comment (lmignon) [reply procurement_auto_create_group/tests/test_auto_create.py]: @rousseldenis done

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by jbaudoux — APPROVED** [review]
LGTM

**Pull request merged**